examples/gameroom/docker-compose.yaml

```diff
@@ -133,24 +163,17 @@ services:
       ports:
         - 8088:8085
       volumes:
-        - key-registry:/key_registry_shared
         - acme-var:/var/lib/splinter
         - ./splinterd-config:/configs
-        - ./node_registry:/node_registry
       entrypoint: |
         bash -c "
-          # We need to wait for the generated key registry to be available
-          while [ ! -f /key_registry_shared/keys.yaml ]; do \
-            echo 'waiting for key registry'; \
-            sleep 1; \
-          done && \
-          # Copy the generated key registry to its expected location
-          cp -a /key_registry_shared/keys.yaml /var/lib/splinter && \
           splinter cert generate --skip && \
           splinterd -c ./configs/splinterd-node-acme.toml -vv \
               --service-endpoint tcp://0.0.0.0:8043 \
               --network-endpoints tcps://0.0.0.0:8044 \
+              --advertised-endpoints tcps://splinterd-node-acme:8044 \
               --bind 0.0.0.0:8085 \
+              --registry http://registry-server:80/nodes.yaml \
```

Comment:
--registry will work but it has been updated to --registries

Reply:
Fixed